node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeFailer.java

```diff
@@ -124,6 +133,25 @@ protected void maintain() {
             failActive(node, reason);
         }
 
+        // Retire active nodes. Only allow 1 active host to be wantToRetire at a time for rate limiting.
+        final long maxWantToRetireHosts = 1;
+        List<Node> candidateNodes = activeNodes.stream()
+                .filter(node -> failAllowedFor(node.type()))
```

Comment:
`failAllowedFor()` may allow a single failure of proxy(host), config(host) and controller(host), but none of those support retiring...

Reply:
Looked at the code again, maybe the do... I thought I saw a configserver in AWS being stuck in retired for a few days, but gone now.